### ISSUE TITLE
Client Warning - non 16x16 texture

### ISSUE THREAD

**Issue by carstorm** [ZeroNoRyouki/BigReactors#29]
Get the following warnings in client log.

[17:44:12] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/On_on.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:12] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/hotFluidIn.png with size 17x17 will have visual artifacts at mip level 4, it can only support level 0. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:12] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/coolantOut.png with size 17x17 will have visual artifacts at mip level 4, it can only support level 0. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:12] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/wasteEject_off.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:12] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/On_off.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/hotFluidOut.png with size 17x17 will have visual artifacts at mip level 4, it can only support level 0. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/wasteManual_off.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/items/turbine/legacy/housingCoreTop.png with size 9x9 will have visual artifacts at mip level 4, it can only support level 0. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/Off_off.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/Off_on.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/wasteEject_on.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/wasteManual_on.png with size 18x18 will have visual artifacts at mip level 4, it can only support level 1. Please report to the mod author that the texture should be some multiple of 16x16.
[17:44:13] [Client thread/WARN]: Texture bigreactors:textures/guiIcons/coolantIn.png with size 17x17 will have visual artifacts at mip level 4, it can only support level 0. Please report to the mod author that the texture should be some multiple of 16x16.

**Comment by ZeroNoRyouki** [comment]
Yeah we are aware of that. We are redoing the old GUI textures but it's a slow-going process
